certmanager/ca.go

```diff
@@ -14,7 +14,71 @@
 
 package certmanager
 
+import (
+	"crypto/rsa"
+	"crypto/x509"
+	"fmt"
+	"time"
+)
+
+const (
+	// The time to live for issued certificate.
+	certTTL = time.Hour
+	// The time to live for self-signed root CA certificate.
+	rootCertTTL = 24 * time.Hour
```

Comment:
Just some numbers that sound reasonable to me (unless you disagree). Later these should be configurable with these as defaults.